data_structures/linked_list/is_palindrome.py

```diff
@@ -1,14 +1,13 @@
 from dataclasses import dataclass
```

Comment:
To remove the quotes around types do:
```suggestion
from __future__ import annotations

from dataclasses import dataclass
```

Reply:
done